src/riscv/lib/src/machine_state/memory/state.rs

```diff
@@ -70,7 +75,10 @@ impl<const PAGES: usize, const TOTAL_BYTES: usize, B, M: ManagerBase>
         E: Elem,
         M: ManagerWrite,
     {
-        let length = E::STORED_SIZE.get();
+        let length = E::STORED_SIZE
+            .try_into()
+            .expect("Elem size `E::STORED_SIZE` must fit into u64");
```

Comment:
Conversions like these happen quite a lot. Perhaps it is best to make `STORED_SIZE` a `u64` as well?